github/validate-comment/README.md

```diff
@@ -0,0 +1,39 @@
+# <!--name-->github/validate-comment<!--/name-->
```

Comment:
From my perspective, this is more of a confirmation than a validation.

But if it's too much of a hassle, we can leave it like that.

Reply:
Actually, I don't know..

I just noticed.. it actually does validate.. but it also does confirm.

Reply:
So, it validates if the comment is from a user with write permissions and if that's okay it confirms it with a thumbs up..

Hence, the name is probably OK.